src/gamegrid.js

```diff
@@ -0,0 +1,9 @@
+let rows = 20;
+let cols = 10;
+let gamegrid = new Array(rows).fill(0).map(() => new Array(cols).fill(0));
+for (let y = 0; y < 20; y++) {
+    for (let x = 0; x < 10; x++) {       
```

Comment:
Perhaps we should use our consts in all cases
```suggestion
for (let y = 0; y < rows; y++) {
    for (let x = 0; x < cols; x++) {       
```

Reply:
we have removed this piece of code and replaced it.